app.py

```diff
@@ -0,0 +1,44 @@
+import requests
+
+class PublicNeeds(object):
+    def __init__(self):
+        pass
+
+    def requesting(self, page=1, limit=1000, last_page=4):
+        '''
+        Kwarg page will be used in for to iterate from this number to last_page
+        Limit is also used with default value of 100 as recommended in api docs.
+        For now i willl use self.requested_pages to append
+        '''
+        requested_pages = []
+        for dictionary in range(page, last_page):
+            req = requests.get('https://api-v3.mojepanstwo.pl/dane/zamowienia_publiczne.json?page={}&limit={}'.format(page, limit)).json()
+            page+=1
+            requested_pages.append(req)
+            return(requested_pages)
```

Comment:
this function will end after one iteration, is it expected behavior?

Reply:
fixed